### ISSUE TITLE
Multiple GPUs for inference 

### ISSUE THREAD

**Issue by rozeappletree** [kukrishna/genaudit#1]
Is there any method available out of the box to run `FactChecker("hf:kundank/genaudit-usb-flanul2")` inference with multiple GPUs? My 24gigs GPU goes OOM.

**Comment by rozeappletree** [comment]
Can we do [distributed inference](https://huggingface.co/docs/accelerate/main/en/usage_guides/distributed_inference) with multiple GPUs? i.e Loading parts of a model onto each GPU.

**Comment by kukrishna** [comment]
Yes, it is possible. We can split the model across more than one GPU if it doesn't fit in one. 
The current version of the library does not support it, but I will add that when I have more bandwidth.
